packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -123,7 +123,14 @@ export class Builder extends EventEmitter {
 
   async run({watch: watchGlobs = false} = {}) {
     const globs = this.getGlobs();
-    const schemaPaths = this.getSchemaPaths();
+    let schemaPaths: string[];
+
+    try {
+      schemaPaths = getGraphQLSchemaPaths(this.config);
+    } catch (error) {
+      this.emit('error', error);
```

Comment:
emit our new error and exit early since we don't have a stable graphql configuration to continue processing.